CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## [0.8.1] - 2024-10-09
```

Comment:
@howard-at-cb also update package.json

Reply:
thank you @jazz-cb , package.json change is covered here https://github.com/coinbase/coinbase-sdk-nodejs/pull/293